### ISSUE TITLE
VoxForge Task - much worse recognition without pitch feature

### ISSUE THREAD

**Issue by iustinaabc** [espnet/espnet#1609]
Hello, 
I am currently working on the VoxForge ASR task and encounter some issues when using only the 80 log-mel filterbank features (no CMVN, no pitch features). The features were extracted with Kaldi script **make_fbank.sh**

I have been training an attention-based RNN with following architecture:

```
# encoder related
etype: vggblstmp     
elayers: 4
eunits: 320 
eprojs: 320 
subsample: "1_2_2_1_1" 

# decoder related
dlayers: 1
dunits: 300 

# attention related
atype: location
adim: 300 
aconv-chans: 10
aconv-filts: 100

# hybrid CTC/attentions
mtlalpha: 0.3 

# minibatch related
batch-size: 35 
maxlen-in: 600  
maxlen-out: 150 

# optimization related
sortagrad: 0 
opt: adadelta
epochs: 25 
patience: 3 

# scheduled sampling option
sampling-probability: 0.5
```

I get error rates of 115% WER and 85 % CER.
The same results occur when I use the same 80 log mel fbanks extracted with Librosa.

Curiously, when I use **make_fbank_pitch.sh** to get **80 log mel-fbanks + pitch** (no CMVN), results are much better: ~19% CER and 42 %WER. I am wondering what goes wrong here, since I read that usually pitch only decreases the error rates by 1-2% at most.

**Comment by kamo-naoyuki** [comment]
I have tested no pitch feats with voxforge with cmvn and got slightly worse result, about 1-2% in that time.
Recently I try same task again and I found there are no difference between w/ and w/o pitch https://github.com/espnet/espnet/blob/v.0.7.0/egs2/voxforge/asr1/README.md.
(though this is espnet2' result), so potentially it can give same performance at voxforge task.

How about trying CMVN? CMVN has significant improvement in some tasks, but I don't know it about voxforge. The other possibility is initialization of parameters.

Would you show loss/acc graph? Then some guys can advise more accurately.

**Comment by iustinaabc** [comment]
> I have tested no pitch feats with voxforge with cmvn and got slightly worse result, about 1-2% in that time.

By _no pitch_ do you mean raw features or fbanks?

> How about trying CMVN? 
> Would you show loss/acc graph? Then some guys can advise more accurately.

These graphs are for **83 feats (fbanks + pitch)** and **CMVN** (_make_fbank_pitch.sh_)
![image](https://user-images.githubusercontent.com/46626614/75109427-2bfbfc80-5623-11ea-9dda-c82ed51605f9.png)

These graphs are for **83 feats (fbanks + pitch)** and **no CMVN** (_make_fbank_pitch.sh_)
![image](https://user-images.githubusercontent.com/46626614/75106694-83986900-561f-11ea-8cb5-43201ecf6062.png)

These graphs are for **80 feats (log mel fbanks)**, **no pitch, no CMVN** (Kaldi's _make_fbank.sh_)
![image](https://user-images.githubusercontent.com/46626614/75106780-408ac580-5620-11ea-9e49-741bb879b751.png)

I have yet to try **80 feats (log mel fbanks)**, no pitch, **with CMVN**

**Comment by iustinaabc** [comment]
Also, forgot to mention that I'm experimenting with the **English Voxforge**, not Italian

**Comment by sw005320** [comment]
Yes, please post it with no pitch, with CMVN, once it is finished. The use of CMVN changes the result a lot.

**Comment by iustinaabc** [comment]
Hi, on second attempt with the same architecture as above and the **Librosa 80 log mel fbank (no CMVN, no Pitch)** features, I got **pretty good results** for decoding: **41 % WER, 17% CER**. I think somewhere in the _run.sh_ I was reading the data from the wrong directories, which messed up the training and results.

I did not train a new network on the same features with CMVN, because in my application I need the simplest, non-normalized features. Anyway, from previous experiments with Kaldi fbank + pitch features, CMVN only brought an improvement in WER/CER of about 1-2% in comparison with non-CMVN features. 

What's still intriguing is that in your latest updated Voxforge results (with RNN), you get WER of ~42%, but CER of ~12%, which is ~8% lower than what I get with my hybrid attention-CTC decoder. 

![image](https://user-images.githubusercontent.com/46626614/75434535-fd975d80-5951-11ea-810b-c3129b85a3af.png)
